backend/sdlbackend/sdl_backend.cpp

```diff
@@ -190,8 +186,19 @@ void igSDLRunLoop(SDL_Window *window, VoidCallback loop, VoidCallback beforeRend
         glViewport(0, 0, (int)io->DisplaySize.x, (int)io->DisplaySize.y);
         glClearColor(sdl_clear_color.x * sdl_clear_color.w, sdl_clear_color.y * sdl_clear_color.w, sdl_clear_color.z * sdl_clear_color.w, sdl_clear_color.w);
         glClear(GL_COLOR_BUFFER_BIT);
+
+        if (beforeRender != NULL)
+        {
+            beforeRender();
+        }
+
         ImGui_ImplOpenGL3_RenderDrawData(igGetDrawData());
 
+        if (afterRender != NULL)
+        {
+            afterRender();
+        }
+
```

Comment:
put this at the end of this `while` loop

Reply:
Ah, I see.  Sorry that this is a rubbish PR - I'll close it.
Is there any appetite to add beforeImFrame/afterImFrame callbacks instead?  In the current implementation (both here and glfw) there appears to be no way to inject other OpenGL calls into the render loop.